markdown_lab/core/converter.py

```
@@ -380,26 +380,33 @@
+
+    def _save_content_chunks(
+        self, 
+        markdown_content: str, 
+        url: str, 
+        output_filename: str, 
+        chunk_dir: str, 
+        chunk_format: str
+    ) -> None:
+        """Save content chunks if chunks are generated successfully."""
+        if chunks := self.create_chunks(markdown_content, url):
+            from markdown_lab.utils.chunk_utils import ContentChunker
```

Pylint (3.3.7)

[convention] Import outside toplevel (markdown_lab.utils.chunk_utils.ContentChunker)

(C0415)